strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/domain/CronTasksConfigurationDto.java

```diff
@@ -15,11 +15,11 @@
         implements Serializable
 {
 
-    private Set<CronTaskConfigurationDto> cronTaskConfigurations = new LinkedHashSet<>();
+    private Set<CronTaskConfigurationDto> cronTaskConfigurations;
 
     public Set<CronTaskConfigurationDto> getCronTaskConfigurations()
     {
-        return cronTaskConfigurations;
+        return cronTaskConfigurations != null ? cronTaskConfigurations : new LinkedHashSet<>();
```

Comment:
it will have a bad result if the `cronTaskConfigurations == null` and someone will do `getCronTaskConfigurations().add(configuration)`.

probably it will be better to do it like this:
```
return cronTaskConfigurations != null ? cronTaskConfigurations : cronTaskConfigurations = new LinkedHashSet<>();
```

Reply:
yeah, thanks :)